<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTableDialog</class>
 <widget class="QDialog" name="CreateTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Table</string>
  </property>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="columnNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Column Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="columnNameInput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="columnType">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addColumnButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Column</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
